picard/tagger.py

```diff
@@ -527,6 +527,10 @@ def add_files(self, filenames, target=None, result=None):
                 file.load(partial(self._file_loaded, target=target))
                 QtCore.QCoreApplication.processEvents()
 
+    def add_file(self, file_, callback):
+        self._pending_files_count += 1
```

Comment:
If this is only used in case the extension load fails, than there is no need to increment the pending files count.

Reply:
Indeed. Looks like we might have a bug here anyway, because in case of error the callback in tagger.py might never get called and the counter never decreases. I'll take a look.

Reply:
See changes in https://github.com/metabrainz/picard/pull/1576/commits/656ac2d097c4d4c5d9a01e7ffcd7ec751afbfd9b, https://github.com/metabrainz/picard/pull/1576/commits/df3c4c941d186fd62d93f7b008369169be2050cf and https://github.com/metabrainz/picard/pull/1576/commits/c303343892105804ff672e3ac6a10a6a1d15c9da

Reply:
Seems fine. Going to test it.

Reply:
Didn't catch anything wrong while testing. :)